lib/fizzy/execute.hpp

```diff
@@ -33,12 +36,30 @@ struct ExecutionResult
     constexpr explicit ExecutionResult(bool success) noexcept : trapped{!success} {}
 };
 
+/// Shortcut for execution that resulted in successful execution, but without a result.
 constexpr ExecutionResult Void{true};
+/// Shortcut for execution that resulted in a trap.
 constexpr ExecutionResult Trap{false};
 
-// Execute a function on an instance.
+/// Execute a function on an instance.
+///
+/// @param instance    The instance, see `instance.hpp`.
+/// @param func_idx    The function index. MUST be a valid index, otherwise undefined behaviour
+///                    (including crash) happens.
+/// @param args        The pointer to the arguments. The number of items must match the expected
+///                    number of input parameters of the function, otherwide underfiend behaviour
```

Comment:
```suggestion
///                    number of input parameters of the function, otherwide undefined behaviour
```

None of you caught this 😅

Reply:
Also codespell thinks `otherwide` and `underfiend` are ok, but catches `undefiend`

Reply:
Yeah.. fixed all.